From 04972fb4ebc98f2b287a246e7a30d3f96ac0b646 Mon Sep 17 00:00:00 2001
From: Eugeniy Paltsev <Eugeniy.Paltsev@synopsys.com>
Date: Fri, 22 Dec 2017 13:46:58 +0300
Subject: [PATCH 30/46] ARC: HSDK: CONFIG: add options to store environment in
 SPI flash

NOTE: It is optional and is disabled by default.

Signed-off-by: Eugeniy Paltsev <Eugeniy.Paltsev@synopsys.com>
---
 include/configs/hsdk.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/include/configs/hsdk.h b/include/configs/hsdk.h
index 687610896c..cdc1b2e365 100644
--- a/include/configs/hsdk.h
+++ b/include/configs/hsdk.h
@@ -60,6 +60,10 @@
  * Environment settings
  */
 #define CONFIG_ENV_SIZE			SZ_16K
+/* Last 64K block is for env */
+#define CONFIG_ENV_OFFSET		(SZ_64K * 16)
+/* 64K block */
+#define CONFIG_ENV_SECT_SIZE		SZ_64K
 
 #define CONFIG_EXTRA_ENV_SETTINGS \
 	"core_dccm_0=0x10\0" \
-- 
2.11.0

